docs-internal/api/nu-designer-openapi.yaml

```diff
@@ -1303,6 +1303,136 @@ paths:
       security:
       - {}
       - httpAuth: []
+  /api/parameters/{scenarioName}/validateAdhoc:
```

Comment:
Maybe `/api/scenarioTesting/{scenarioName}/adHoc/validate` and similar for `/parameters` and `/invoke`

Reply:
I've changed only this endpoint to `api/scenarioTesting/{scenarioName}/adhoc/validate` for now. It is because  `/testParameters` is in `api/testInfo` path and _invoke_ (`testWithParameters`) is in `api/processManagement`. I am not sure that it should be all under `/scenarioTesting` right now. If you have some thoughts/plans about that then we can talk about that. WDYT?

Reply:
It would be nice to have all resources that are used in this flow in one path. For now at least:
- `/testInfo/testParameters`
- `/processManagement/testWithParameters`

But in some longer perspective, all resources nested in `/testInfo` and `/processManagement/*test*`. Do you have :muscle: and :clock1:  to move them?